doc-content/enterprise-only/installation/git-import-proc.adoc

```diff
@@ -1,5 +1,5 @@
 [id='git-import-proc']
-
+//Comment - Mark for deletion, duplicate of project-data/git-import-project.adoc 
```

Comment:
Is this for 7.4? Because that mark for deletion is currently at master but not at prod

Reply:
This is a duplicated doc that I would like to remove but retaining it for now just incase it is needed elsewhere.